src/bn_pending_txns.erl

```diff
@@ -24,9 +24,10 @@
     | #blockchain_txn_security_exchange_v1_pb{}
     | #blockchain_txn_stake_validator_v1_pb{}
     | #blockchain_txn_unstake_validator_v1_pb{}
-    | #blockchain_txn_transfer_validator_stake_v1_pb{}.
+    | #blockchain_txn_transfer_validator_stake_v1_pb{}
+    | #blockchain_txn_state_channel_open_v1_pb{}.
 
--type nonce_type() :: none | balance | gateway | security.
+-type nonce_type() :: none | balance | gateway | security | dc_nonce.
```

Comment:
Rename to just `dc` since we don't say security_nonce  either

Reply:
Fair enough